templates/web/common/site_layout.tt.html

```diff
@@ -367,9 +367,9 @@ <h3 class="title-5 text-medium">[% lang('footer_join_the_community') %]</h3>
 						<p>[% f_lang('f_join_us_on_slack', { url => "https://slack.openfoodfacts.org" } ) %]</p>
 						<p><a href="https://forum.openfoodfacts.org/">Forum</a></p>
 						<p id="footer_social_icons">[% f_lang('f_footer_follow_us_links', { links => '
-							<a href="https://x.com/' _ twitter_account _ '"><img src="/images/icons/dist/x.svg" class="footer_social_icon" alt="X"></a>
+							<a href="https://www.x.com/' _ twitter_account _ '"><img src="/images/icons/dist/x.svg" class="footer_social_icon" alt="X"></a>
```

Comment:
For X the canonical url is indeed https://x.com so you didn't have to change it !

```suggestion
							<a href="https://x.com/' _ twitter_account _ '"><img src="/images/icons/dist/x.svg" class="footer_social_icon" alt="X"></a>
```